docs/RepoToolset.md

```diff
@@ -99,7 +99,7 @@ Versions.props file is present in the repo and defines versions of all dependenc
     <UsingToolIbcOptimization>true</UsingToolIbcOptimization>
   
     <!-- RepoToolset package version -->
-    <RoslynToolsMicrosoftRepoToolsetVersion>1.0.0-alpha27</RoslynToolsMicrosoftRepoToolsetVersion>
+    <RoslynToolsRepoToolsetVersion>1.0.0-alpha27</RoslynToolsRepoToolsetVersion>
```

Comment:
:memo: The version itself seems to be old. Not sure if it needs to be updated now or later.